gravityflow/gravityflowformconnector
# Fix compatibility issue with the new Stripe Checkout

re:[HS#10840](https://secure.helpscout.net/conversation/953652183/10840?folderId=2308452)

This PR fixes an issue where the form submission step cannot be released if the payment method set in the target form is Stripe Checkout (Stripe add-on > 3.0).

We resolve this issue by changing the hook priority of `gform_after_submission` to `40`, because Stripe Checkout set its redirect scripts with the same hook and the priority is `50`. We use a lower number to make sure our function would run before the redirection.

## Testing instructions
1. Follow the testing instructions in the ticket to confirm the issue exists.
2. Switch to this branch, the issue should be resolved.
